examples/tutorial/tutorial_part_one.ipynb

```diff
@@ -148,7 +148,7 @@
     "# clipper_manager must be on your path:\n",
     "import sys\n",
     "import os\n",
-    "sys.path.append(os.path.abspath('../../management/'))\n",
+    "sys.path.append(os.path.abspath('../../clipper_admin/'))\n",
```

Comment:
Do you think we should import the module directly like this or make them install the pip package?